tools/lock_inspect/evm_connection.rb

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env ruby
+
+require 'pg'
+require 'yaml'
+require 'optparse'
+
+def error_exit(e, conn, error_message = '', exit_code = 1)
+  puts e.message
+  puts error_message unless error_message.empty?
+  conn.close if conn
+  exit(exit_code)
+end
+
+def process_app_name(app_name)
+  if app_name.end_with?('..')
+    raise "The application name for MIQ worker: {#app_name} is truncated"
```

Comment:
Should we probably capture whatever we can instead of raising here.  Leave the missing data as nil.